Task8/src/api/video.js

```diff
@@ -0,0 +1,83 @@
+import { ActionType } from '../store/actionTypes';
+
+const API_KEY = 'AIzaSyALMiCr_df0KxwdiWxpGeQBSCrczG5RcRs';
```

Comment:
API key should be stored in separate config file with secrets and shouldn't be included in repo

Reply:
Yes, ideally all such requests with secret API keys should go through back-end part.

Reply:
Deleted